### PR TITLE
Fix number_from_string() function in cnex

### PR DESCRIPTION
**Pull request by gitlarryf**

### CODE REVIEW

**Comment by ghewgill** [comment]
This also needs to include the corresponding test.

**Review by ghewgill — CHANGES_REQUESTED** [review]
This also needs to include the corresponding test.

File: exec/cnex/number.c [hunk 1]
```diff
@@ -154,6 +154,55 @@ void number_toString(Number x, char *buf, size_t len)
 
 Number number_from_string(char *s)
```
Comment (ghewgill) [inline exec/cnex/number.c:155]: This should be `const char *`.

**Review by ghewgill — CHANGES_REQUESTED** [review]

File: exec/cnex/number.c (hunk 1 above)
Comment (gitlarryf) [reply exec/cnex/number.c]: Changed.

**Pull request merged**